app/views/decidim/challenges/challenges/_challenge.html.erb

```diff
@@ -1 +1,3 @@
-<%= translated_attribute(challenge.title) %>
+<div class="column">
+    <%= cell "decidim/challenges/challenge_list_item", challenge %>
```

Comment:
Why are you implementing and using a challenge_list_item cell?
You've already created the challenge_m cell that is not used anywhere and should be used here instead